src/Aspire.Dashboard/Components/Controls/ResourceDetails.razor

```diff
@@ -41,86 +41,59 @@
-                            MaxDisplayLength="0">
-                            <ContentAfterValue>
-                                @if (context.Url != null)
-                                {
-                                    <a href="@context.Url" target="_blank">@context.Url</a>
-                                }
-                                else
-                                {
-                                    @context.Text
-                                }
-                            </ContentAfterValue>
-                        </GridValue>
-                    </AspireTemplateColumn>
-                </FluentDataGrid>
+                <PropertyGrid
+                    TItem="DisplayedEndpoint"
+                    Items="@FilteredEndpoints"
+                    ContentAfterValue="(vm) => GetContentAfterValue(vm)"
+                    ValueSort="_endpointValueSort"
+                    IsValueMaskedChanged="@OnValueMaskedChanged"
+                    HighlightText="@_filter"
+                    GridTemplateColumns="1fr 1.5fr" />
             </FluentAccordionItem>
             <FluentAccordionItem Heading="@ControlStringsLoc[nameof(ControlsStrings.ResourceDetailsEnvironmentVariablesHeader)]" Expanded="true">
                 <div slot="end">
                     <FluentBadge Appearance="Appearance.Neutral" Circular="true">
-                        @FilteredItems.Count()
+                        @FilteredEnvironmentVariables.Count()
                     </FluentBadge>
                 </div>
-                <PropertyGrid TItem="EnvironmentVariableViewModel"
-                              Items="@FilteredItems"
-                              NameColumnValue="(vm) => vm.Name"
-                              ValueColumnValue="(vm) => vm.Value"
-                              NameSort="_nameSort"
-                              ValueSort="_valueSort"
-                              EnableValueMasking="true"
-                              GetIsItemMasked="(vm) => vm.IsValueMasked"
-                              SetIsItemMasked="(vm, newValue) => vm.IsValueMasked = newValue"
-                              IsMaskedChanged="@CheckAllMaskStates"
-                              HighlightText="@_filter"
-                              GridTemplateColumns="1fr 1.5fr" />
+                <PropertyGrid
+                    TItem="EnvironmentVariableViewModel"
+                    Items="@FilteredEnvironmentVariables"
+                    IsValueMaskedChanged="@OnValueMaskedChanged"
+                    HighlightText="@_filter"
+                    GridTemplateColumns="1fr 1.5fr" />
             </FluentAccordionItem>
         </FluentAccordion>
     </div>
 </div>
+
+@code {
+    private static RenderFragment GetContentAfterValue(DisplayedEndpoint vm)
+    {
+        if (vm.Url is null)
+        {
+            return @<span>@vm.Text</span>;
+        }
+        else
+        {
+            return @<a href="@vm.Url">@vm.Text</a>;
```

Comment:
```suggestion
            return @<a href="@vm.Url" target="_blank">@vm.Text</a>;
```

Reply:
Previously the text in the `<a>` tag was the Url property. Is Text always the same value as Url?

https://github.com/dotnet/aspire/blob/3439c433770255f06ef286c1663255f066687324/src/Aspire.Dashboard/Components/Controls/ResourceDetails.razor#L94

Reply:
The new code displays `Text` only when it differs from `Url`, and includes the `Url` in the `title`, so that it appears in a tool tip.